consensus/polybft/checkpoint_manager_test.go

```diff
@@ -39,83 +39,109 @@ func TestCheckpointManager_SubmitCheckpoint(t *testing.T) {
 
 	validators := validator.NewTestValidatorsWithAliases(t, aliases)
```

Comment:
Could it lead to race conditions, since both subtests are using the same validators slice (though probably they are just reading them), but still think it would be a good idea that each one has its own copy. Wdyt?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1982/commits/c4720038620cb297c87aca3d93003e334f237070